Makefile

```diff
@@ -48,7 +48,7 @@ ROX_IMAGE_FLAVOR ?= $(shell \
 	  echo "development_build"; \
 	fi)
 
-DEFAULT_IMAGE_REGISTRY := quay.io/stackrox-io
```

Comment:
You can still override `DEFAULT_IMAGE_REGISTRY` when it's set the old way:
```bash
make DEFAULT_IMAGE_REGISTRY=quay.io/blah/fo-bar my-target
# or
make my-target DEFAULT_IMAGE_REGISTRY=quay.io/blah/fo-bar
```
I.e. use make variable, not env variable.

What is your use-case, do you do it to be able to push containers in your private repos?

Reply:
Yes, this is my use-case. TIL that these do not have the same behavior :
- `make DEFAULT_IMAGE_REGISTRY=quay.io/blah/fo-bar my-target`
- `DEFAULT_IMAGE_REGISTRY=quay.io/blah/fo-bar make my-target`

While they are equivalent when using `?=`

Reply:
Thank you for the review @msugakov .
I reverted this part of the PR, to keep only the docker envvar.